### DISCUSSION TITLE
Zscaler in use and blocking atom:// style urls for packages

### DISCUSSION THREAD

**Discussion by jason0** [pulsar-edit/.github#290] [Community Packages]
Hello,

I just switched over to pulsar 1.119.0.  I am unable to install any community packages because zscaler is effectively blocking urls of type atom://.

How do I download packages and install them?  

Thank  you for your time!

--jason

**Comment by savetheclocktower** [comment]
You can install packages within Pulsar by going to Settings -> Install.

If you've set up `ppm` on the command line, you can also do `ppm install [name-of-package]`.

**Reply by jason0** [reply]
I did try going to settings->install and it just says "unable to get local issuer certificate" which is zscaler stepping on it.

I will look into ppm, (perl package manager?) 

I probably should have mentioned this is a windows 10 system.

**Reply by savetheclocktower** [reply]
> I will look into ppm, (perl package manager?)

No, this is Pulsar Package Manager — it ships with Pulsar. If you upgrade to the just-released Pulsar 1.121, the installer will offer you the option of adding `pulsar` and `ppm` to your PATH so you can invoke them from the command line. (But if installing packages from within Pulsar doesn't work, then `ppm` probably won't work either.)

> I did try going to settings->install and it just says "unable to get local issuer certificate" which is zscaler stepping on it.

That's odd. We've got SSL certificates and everything. Zscaler is acting pretty overzealously.

The “scariest” thing about Pulsar is that we don't sign the Windows version, so I'm a bit surprised that something so draconian has allowed you to install it on your machine. Past that, though, I'm not sure what would need to happen. I'm assuming this is a work machine? Does your company's IT have any guidance here?
